### DISCUSSION TITLE
Possible error in Script Details Wiki page?

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#57] [Q&A] [answered]
In the Wiki page for Script Details (https://github.com/gitbls/sdm/wiki/Script-Details), it says "If --user is specified ...".  I believe --user MUST be specified.  When I tried to customize an image without doing so (since I was going to specify user and password in the burn step) I got an error message saying was required.

**Comment by gitbls** [comment, marked as answer]
Ah, but the `--nouser` switch can be used to customize an IMG without creating a user.

**Reply by dshaw619** [reply]
Thanks for pointing that out.  I hadn't remembered about --nouser since the description says 'Primarily for use with sdm-gburn'.  This will simplify my workflow (enter unique password per host interactively during burn).

**Reply by gitbls** [reply]
I updated the documentation for --user to say that it's useful if you want to late-create the user, for instance at burn time or with sdm-gburn.
